### DISCUSSION TITLE
How can I get real-time outputs from TTS model?

### DISCUSSION THREAD

**Discussion by MekanJuma** [coqui-ai/TTS#2974] [General Q&A]
I have been researching this for a long time, but there is no clear and exact answer. I want to be able to get Real-Time results, yeah it depends on where you deployed whether it is GPU or not, but even if I run this in GPU, it takes some time to get output. I would appreciate your help guys!

**Comment by erogol** [comment]
you can use faster models like VITS

**Reply by MekanJuma** [reply]
Did you try this before, is there any resource that you can recommend to try this? Thank you bro for your response again!

**Reply by erogol** [reply]
No problem bro. You can check https://tts.readthedocs.io/en/latest/models/tortoise.html
